README.md

```diff
@@ -8,6 +8,8 @@
   <a href="https://goreportcard.com/report/github.com/kubermatic/kubeone">
     <img src="https://goreportcard.com/badge/github.com/kubermatic/kubeone" alt="Go Report Card" />
   </a>
+  <a href="https://bestpractices.coreinfrastructure.org/projects/1189"><img src="https://bestpractices.coreinfrastructure.org/projects/1189/badge">
```

Comment:
This should link to the KubeOne page, not to the Fluentd page.

Reply:
Sorry, fixed